scripts/tools/PROG.rst

```diff
@@ -568,6 +568,53 @@ The following pseudo-code demonstrates a basic sequence for metric calculation a
            free(phMetrics);
        }
 
+Calculating Multiple Metrics
+-----------
+
+${t}MetricGroupCalculateMultipleMetricValues can be used to calculate one or more sets of metric values from raw data.
+Sample below shows how to calculate and process multiple metric values.
+
+.. parsed-literal::
+
+       // Get data count and total metric count
+       uint32_t setCount = 0;
+       uint32_t metricValueCount = 0;
+       ${t}MetricGroupCalculateMultipleMetricValues(hMetricGroup, type, rawDataSize, pRawData, &setCount, &metricValueCount, nullptr, nullptr);
+
+       // Get metric counts and metric values
+       std::vector<uint32_t> metricCountPerSet(setCount);
+       std::vector<${t}_typed_value_t> metricValues(metricValueCount);
+       ${t}MetricGroupCalculateMultipleMetricValues(hMetricGroup, type, rawDataSize, pRawData, &setCount, &metricValueCount, metricCountPerSet.data(), metricValues.data());
```

Comment:
Can we add a line where we are modifying the value for metricCountPerSet.
@matcabral 
Should the below be OK ?
metricCountPerSet[0] = metricValueCount / 2;
metricCountPerSet[1] = metricValueCount / 2;

Reply:
I don't expect the use case of limiting metrics per set to be very common. So, IMHO adding it in the example will unnecessarily complicate the common use case.